core/templates/domain/collection/collection-node.model.ts

```diff
@@ -24,12 +24,12 @@ import { LearnerExplorationSummaryBackendDict } from
 
 export interface CollectionNodeBackendDict {
   'exploration_id': string;
-  'exploration_summary': LearnerExplorationSummaryBackendDict;
+  'exploration_summary': LearnerExplorationSummaryBackendDict | null;
```

Comment:
You are adding quite a few new `null` values, is there a way to limit the number of those? Like changing the code in a way that disallows so many nulls?